### DISCUSSION TITLE
Does zettlr visualize images from obsidian like: ![[attachments/Pasted image 20231231021529.png]]

### DISCUSSION THREAD

**Discussion by tyoc213** [Zettlr/Zettlr#5367] [Q&A]
I have been using obsidian, but I want to use zettlr and some of my notes have pasted images like `![[attachments/Pasted image 20231231021529.png]]` which are not displayed inline

Is there a way to visualize them? or should I replace this format on all my notes?

**Comment by hittjwX** [comment]
Supporting Obsidian-style images would be excellent. Otherwise, you'll have to convert the image into a Markdown format `![alt text](Isolated.png "Title")` which Obsidian supports.

**Comment by tyoc213** [comment]
Just to add, I saw now this on my Obsidian settings :)

<img width="630" alt="image" src="https://github.com/user-attachments/assets/69190fa2-702e-46a8-b3d1-ff9c819182fa">
